BMM.UI.iOS/Application/ViewController/EditTrackCollectionViewController.cs

```diff
@@ -27,7 +27,11 @@ public override void ViewDidLoad()
         {
             base.ViewDidLoad();
 
-            NavigationController.PresentationController.Delegate = new EditTrackCollectionPresentationControllerDelegate(ViewModel);
+            NavigationController.PresentationController.Delegate = new CustomUIAdaptivePresentationControllerDelegate()
+            {
+                OnDidDismiss = HandleDismiss,
+                OnDidAttemptToDismiss = HandleDismiss
```

Comment:
Here it shouldn't invalidate the delegate actions. With the current implementation the dialog is just shown once when trying to dismiss.

Reply:
Sorry, I don't get it. Could you elaborate more?

Reply:
In the edit view after a change the first time you try to drag the sheet down it shakes and asks you if you want to keep editing. With this change it's just happening once. When I try to drag the sheet down again it's not asking if you want keep editing or discard the changes. It should ask you every time if you have changes if you want to keep editing or discard.

Reply:
Ok, now I see. I haven't noticed "not" word in your github comment ;) https://github.com/bcc-code/bmm-app/pull/37#discussion_r661233186

Reply:
you never addressed Oliver's comment